### ISSUE TITLE
After installing 1.0.111, weird behavior at first

### ISSUE THREAD

**Issue by sburkett** [Eugeny/tabby#2569]
I updated to 1.0.111, and upon restarting Terminus, it initially opened with a tab for my last terminal session, though it was totally blank, and with no actual connection. Not the black screen blank, but just the transparent color. I closed the tab, restarted Terminus, and the same thing happened. I run Terminus in full screen, and I noticed that when I resized the window, closed Terminus, then restarted it, the problem went away. Just a heads up. Sorry, didn't think to take a screenshot!

**Comment by sburkett** [comment]
Just upgraded to 1.0.112 and the same behavior occurred. Got a screenshot. Nothing really to see, just a tab for my last terminal that was open before the upgrade.

![image](https://user-images.githubusercontent.com/2219483/83280083-207d1c00-a1a4-11ea-962c-e94f5f331f9d.png)

This time, I tried a different fix. I closed the blank tab, which brought up the default Terminus main menu window automatically. Restarted Terminus and it worked fine.
